README.md

```diff
@@ -45,34 +45,39 @@ To run a single module, specifying the AMQP server, you might use something more
 Quickstart
 ----------
 
-Install RabbitMQ from https://rabbitmq.com/download.html. It's required to run
-tests.
-
-Install libtool-ltdl dev libraries, which are required for Boulder's PKCS11
-support.
+Boulder requires an installation of RabbitMQ, libtool-ltdl, and
+MariaDB 10 to work correctly.
 
 Ubuntu:
-`sudo apt-get install libltdl3-dev`
+
+    sudo apt-get install libltdl3-dev mariadb-server rabbitmq-server
```

Comment:
When I tried this on Ubuntu 14.10, the rabbitmq-server was too old and didn't work. Add a "NOTE: You may need to install the latest RabbitMQ from https://rabbitmq.com/download.html if you get errors."